app/src/main/res/menu/menu_album.xml

```diff
@@ -45,4 +45,14 @@
         android:id="@+id/delete"
         android:title="@string/delete_item" />
 
+    <item
+        android:id="@+id/go_to"
+        android:title="@string/go_to">
+        <menu>
```

Comment:
I'm not sure if having a submenu with only one item in it is the best thing from a UX perspective, but it does keep this consistent with other areas of the app.